clinica/iotools/converters/adni_to_bids/adni_utils.py

```diff
@@ -859,7 +859,7 @@ def create_adni_scans_files(conversion_path, bids_subjs_paths):
         sessions_paths = glob(path.join(bids_subj_path, "ses-*"))
         for session_path in sessions_paths:
             session_name = session_path.split(os.sep)[-1]
-            viscode = session_to_viscode(session_name[4::])
+            viscode = session_label_to_viscode(session_name[4::])
```

Comment:
I'd suggest the following:

```python
session_label = session_id[4::]
viscode = session_label_to_viscode(session_label)
```

I wonder why you're not doing a `session_id_to_viscode` directly which takes care of stripping the `ses-` prefix and converting to viscode?